src/librustc_mir/const_eval/machine.rs

```diff
@@ -56,6 +57,32 @@ impl<'mir, 'tcx> InterpCx<'mir, 'tcx, CompileTimeInterpreter<'mir, 'tcx>> {
         self.dump_place(*dest);
         return Ok(true);
     }
+
+    /// "Intercept" a function call to a panic-related function
+    /// because we have something special to do for it.
+    /// Returns `true` if an intercept happened.
```

Comment:
Doesn't this return an `Err` if an intercept happens?

Reply:
Hm... I guess there used to be (?) cases where it returned `Ok(true)` to indicate "nothing wrong, and also the function call is done", but we don't use that any more so I'll remove that.